src/haddock/modules/analysis/alascan/__init__.py

```diff
@@ -1,21 +1,21 @@
 """
 HADDOCK3 module for alanine scan.
 
-This module is responsible for the alanine (or any other residue) scan analysi
-of the models generated in the previous step of the workflow.
-For each model, the module will mutate the interface residues and calculate
-the energy differences between the wild type and the mutant, thus providing a
-measure of the impact of such mutation. Such difference (delta_score) is always
+This module is responsible for the alanine (or any other residue) scan analysis
+of the model(s) generated in the previous step of the workflow.
+For each model, this module will mutate the interface residues and calculate
+the haddock score differences between the wild type and the mutant, thus providing
```

Comment:
Not only the haddock score is displayed but also the components
```suggestion
the energy components and haddock score differences between the wild type and the mutant, thus providing
```